api/test/docker/manager-api-conf.json

```diff
@@ -0,0 +1,31 @@
+{
+    "conf": {
+      "syslog": {
+        "host": "127.0.0.1"
+      },
+      "listen": {
+        "host": "0.0.0.0",
+        "port": 8080
+      },
+      "dag-lib-path": "",
+      "etcd": {
+        "endpoints": "172.16.238.10:2379,172.16.238.11:2379,172.16.238.12:2379"
+      }
+    },
+    "authentication": {
+      "session": {
+        "secret": "secret",
+        "expireTime": 3600
+      },
+      "user": [
+        {
+          "username": "admin",
+          "password": "admin"
+        },
+        {
```

Comment:
two problems:

1. `user` -> `users`: I will create a new issue to fix this.
2. why do we need two users here?

Reply:
support multiple user to login, I got it.